test/e2e/rollback_test.go

```diff
@@ -61,7 +61,7 @@ var _ = Describe("rollback", func() {
 	})
 	// This spec is done only at first node since policy has to be different
 	// per node (ip addresses has to be different at cluster).
-	Context("when connectivity to default gw is lost after state configuration", func() {
+	Context("[test_id:3793]when connectivity to default gw is lost after state configuration", func() {
```

Comment:
This should be at `It`. This was a bug on the master PR

Reply:
ok, done.